doc/core-algorithms/index.html.spt

```diff
@@ -18,7 +18,10 @@ for line in open(srcfile):
 <p>Aspen's request-processing algorithm is implemented
 with a module called <a
 href="http://algorithm-py.readthedocs.org/"><code>algorithm</code></a>. If you
-want to modify Aspen's behavior you should read up on the <a
+want to modify
+<a href="https://github.com/gratipay/aspen-python/blob/0.40/aspen/website.py">
```

Comment:
I don't think we want to point at the 0.40 version of `website.py`.  How about `https://github.com/gratipay/aspen-python/blob/master/aspen/website.py` instead?